featurepack/core/pom.xml

```diff
@@ -322,5 +322,4 @@
             </plugin>
         </plugins>
     </build>
-
```

Comment:
You can squash this kind of changes in the previous commits, no need to add them in a follow up commit